ext/nnstreamer/tensor_decoder/tensordec-flatbuf.cc

```diff
@@ -172,7 +172,7 @@ fbd_decode (void **pdata, const GstTensorsConfig *config,
   if (gst_buffer_get_size (outbuf) == 0)
     gst_buffer_append_memory (outbuf, out_mem);
   else
-    gst_memory_unref (out_mem);
+    gst_buffer_replace_all_memory (outbuf, out_mem);
```

Comment:
I was wondering when ```out_mem``` is freed, but it's done inside the ```_replace_memory()```,
thank you.!! :smile: